src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -471,8 +502,27 @@ def load_command_table(self, _):
 
     with self.command_group('account management-group subscription', resource_managementgroups_subscriptions_sdk, client_factory=cf_management_group_subscriptions) as g:
         g.custom_command('add', 'cli_managementgroups_subscription_add')
+        g.custom_show_command('show', 'cli_managementgroups_subscription_show')
+        g.custom_command('show-sub-under-mg', 'cli_managementgroups_subscription_show_sub_under_mg')
         g.custom_command('remove', 'cli_managementgroups_subscription_remove')
 
+    with self.command_group('account management-group entities', resource_managementgroups_entities_sdk, client_factory=cf_management_group_entities) as g:
+        g.custom_command('list', 'cli_managementgroups_entities_list')
+
+    with self.command_group('account management-group hierarchy-settings', resource_hierarchy_settings_sdk, client_factory=cf_hierarchy_settings) as g:
+        g.custom_command('list', 'cli_hierarchy_settings_list')
+        g.custom_command('create', 'cli_hierarchy_settings_create')
+        g.custom_command('delete', 'cli_hierarchy_settings_delete')
```

Comment:
Do we need to consider add `confirmation=True` for the delete command?

Reply:
I added it